python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -148,6 +148,9 @@ def __init__(
         self._dome_az_in_position: typing.Union[None, asyncio.Event] = None
         self._dome_el_in_positio: typing.Union[None, asyncio.Event] = None
 
+        self.az_delta = 0.1  # A small move to un-park the Dome
```

Comment:
so, instead of relying on the comment (that is only really here) to specify what the attribute is, use a more expressive name. Something like;

```
# Small offset (in degrees) applied to the dome to unpark it.
self.dome_az_unpark_offset = 0.1
```

Would be better.